linera-core/src/updater.rs

```diff
@@ -323,12 +343,13 @@ where
             let storage = self.local_node.storage_client();
             let certs = storage.read_certificates(keys.into_iter()).await?;
             for cert in certs {
-                self.send_certificate(cert, delivery).await?;
+                self.send_confirmed_certificate(cert, delivery).await?;
             }
         }
         if let Some(cert) = manager.timeout {
             if cert.inner().chain_id == chain_id {
-                self.send_certificate(cert, CrossChainMessageDelivery::NonBlocking)
+                self.remote_node
+                    .handle_optimized_certificate(&cert, delivery)
```

Comment:
Can you introduce the `send_timeout_certificate` method which does just this? it's a "tribe knowledge" right now _why_ timeouts are handled differently.

Reply:
Is it worth having such a method at all? I'd prefer to just add a comment here.

Reply:
Ah, and I should remove the `optimized` here!

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3006/commits/6ac957b6919b273df5ebe6c15ce2e069645b6bf5.